Python 3.13.0 (tags/v3.13.0:60403a5, Oct  7 2024, 09:38:07) [MSC v.1941 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
================ RESTART: C:/Users/HP/OneDrive/Desktop/fileEx.py ===============
Traceback (most recent call last):
  File "C:/Users/HP/OneDrive/Desktop/fileEx.py", line 1, in <module>
    f=open("f1.txt","W")
ValueError: invalid mode: 'W'
>>> 
================ RESTART: C:/Users/HP/OneDrive/Desktop/fileEx.py ===============
Traceback (most recent call last):
  File "C:/Users/HP/OneDrive/Desktop/fileEx.py", line 1, in <module>
    f=open("f1.txt","W")
ValueError: invalid mode: 'W'
>>> 
================ RESTART: C:/Users/HP/OneDrive/Desktop/fileEx.py ===============
Traceback (most recent call last):
  File "C:/Users/HP/OneDrive/Desktop/fileEx.py", line 1, in <module>
    f=open("f1.txt",'W')
ValueError: invalid mode: 'W'
